{
	"id": "ELSA-2020-3185",
	"advisories": [
		{
			"id": "ELSA-2020-3185",
			"title": "ELSA-2020-3185:  python-pillow security update (IMPORTANT)",
			"description": "[5.1.1-12]\n- Fix for CVE-2020-5313\nResolves: rhbz#1789532\n\n[5.1.1-11]\n- Fix for CVE-2020-11538\nResolves: rhbz#1852814",
			"severity": [
				{
					"type": "vendor",
					"source": "linux.oracle.com/security",
					"vendor": "IMPORTANT"
				}
			],
			"references": [
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/cve/CVE-2020-11538.html"
				},
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/cve/CVE-2020-5313.html"
				},
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/errata/ELSA-2020-3185.html"
				}
			],
			"published": "2020-07-29T00:00:00Z"
		}
	],
	"vulnerabilities": [
		{
			"id": "CVE-2020-11538",
			"references": [
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/cve/CVE-2020-11538.html"
				}
			]
		},
		{
			"id": "CVE-2020-5313",
			"references": [
				{
					"source": "linux.oracle.com/security",
					"url": "https://linux.oracle.com/cve/CVE-2020-5313.html"
				}
			]
		}
	],
	"detection": [
		{
			"ecosystem": "oracle:8",
			"criteria": {
				"operator": "OR",
				"criterions": [
					{
						"vulnerable": true,
						"package": {
							"name": "python3-pillow",
							"architectures": [
								"aarch64",
								"x86_64"
							]
						},
						"affected": {
							"type": "rpm",
							"range": [
								{
									"lt": "0:5.1.1-12.el8_2"
								}
							],
							"fixed": [
								"0:5.1.1-12.el8_2"
							]
						}
					}
				]
			}
		}
	],
	"data_source": "oracle"
}
